app/assets/stylesheets/application.scss

```diff
@@ -234,3 +234,17 @@ tr input[type=number] {
   max-height: 120px;
   min-height: 90px;
 }
+
+.chip {
+  position: absolute;
+  top: 12px;
+  right:8px;
+  font-size: 10px;
+  height: 16px;
+  width: 18px;
+  text-align: center;
+  padding-left: 6px;
+  line-height: 16px;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.